.gitignore

```diff
@@ -38,3 +38,7 @@ ENV/
 # Memory dump files
 *.dmp
 *.vmem
+*.img
```

Comment:
Looks good, but let's cap it at `img` and `dmp`, otherwise this list will grow to everybody's personal way of naming files.  There's no standard, but we also can't support everybody's naming scheme.